src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/Linker.cs

```diff
@@ -114,6 +113,7 @@ static Pipeline CreatePipeline (LinkerOptions options)
 			// monodroid tuner steps
 			if (!string.IsNullOrWhiteSpace (options.ProguardConfiguration))
 				pipeline.AppendStep (new GenerateProguardConfiguration (options.ProguardConfiguration));
+			pipeline.AppendStep (new StripEmbeddedLibraries ());
```

Comment:
Maybe this should happen where the `RemoveLibraryResourceZip` step was.

The comments confused me:
```
// end monodroid specific
// monodroid tuner steps
```

Reply:
It should be fine here. At the end we already know all the actions and thus we know which assemblies will be deleted.